.gitignore

```diff
@@ -3,3 +3,9 @@
 dev.html
 jsdoc.json
 /.idea
+node_modules
+dist
```

Comment:
Don't exclude `dist` from the git repo, as it is used by bower.

When you install with bower you should already have builded library without running any additional scripts